__std__/flake.nix

```diff
@@ -16,17 +16,21 @@
   # { inputs, cell }: inputs.nixpkgs, inputs.haskell-nix
   inputs = {
     nixpkgs = {
-      url = "github:NixOS/nixpkgs/34e4df55664c24df350f59adba8c7a042dece61e";
+      url = "github:NixOS/nixpkgs";
     };
     std = {
       url = "github:divnix/std";
       inputs.nixpkgs.follows = "nixpkgs";
     };
     haskell-nix = {
-      url = "github:input-output-hk/haskell.nix/e36a34c4abc02287ec79bf88a0c993f6b66f9945";
+      url = "github:input-output-hk/haskell.nix";
+      inputs = {
```

Comment:
I had tried the exact same myself i.e. to move the `haskell-nix-bootstrap` logic of `__std__/nix/toolchain/library/pkgs.nix` into the flake inputs like you did, but it didn't seem to work, it was triggering a full rebuild of everything...
It is not doing it now? Ill go anche check myself

Reply:
Turns out I must have done something wrong, this works.